plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapAuthenticatorFactory.java

```diff
@@ -39,7 +39,7 @@ public PasswordAuthenticator create(Map<String, String> config)
                 binder -> {
                     configBinder(binder).bindConfig(LdapConfig.class);
                     binder.bind(LdapAuthenticator.class).in(Scopes.SINGLETON);
-                    binder.bind(LdapAuthenticatorClient.class).to(JdkLdapAuthenticatorClient.class).in(Scopes.SINGLETON);
+                    binder.bind(LdapClient.class).to(JdkLdapClient.class).in(Scopes.SINGLETON);
```

Comment:
I think it has to be a part of this commit. https://github.com/trinodb/trino/pull/10116/commits/d3f7eff6991c79e94be1db675aa5fe963374ef8c